tests/typechecker/return_if_prongs.jakt

```diff
@@ -1,11 +1,11 @@
-function foo() -> i32 {
-    if true {
-        return 123
+function max(a: i32, b: i32) -> i32 {
```

Comment:
Please add a separate test instead of complicating this one

Reply:
Alright, I added another test too to ensure the definite returns still work because those didn't have a test yet.